snuba_sdk/orderby.py

```diff
@@ -28,12 +28,18 @@ def validate(self) -> None:
 
 @dataclass(frozen=True)
 class LimitBy(Expression):
-    column: Column
+    columns: Sequence[Column]
     count: int
 
     def validate(self) -> None:
-        if not isinstance(self.column, Column):
-            raise InvalidExpressionError("LimitBy can only be used on a Column")
+        if (
+            not isinstance(self.columns, Sequence)
+            or len(self.columns) < 1
+            or not isinstance(self.columns[0], Column)
```

Comment:
Given that this logic is in two places, I think I want to add a shared

`is_list_of_type(value, type, min_length=1)`

Do you think query_validation.py is the right place for this or should it go in a new file since it's not concerned with validation at the query level?

Reply:
I would put it in expressions.py I think. There are already some type validating functions there.

Reply:
I actually didn't end up doing this because the array join validation and the limit by validation are sort of awkwardly separated in the call-stack and I encountered a circular import for `Column` trying to put this in expressions.